accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlConfigReader.java

```diff
@@ -286,7 +286,10 @@ private void setupAceBean(final String principal,
                 currentAceDefinition, ACE_CONFIG_PROPERTY_PRIVILEGES));
         tmpAclBean.setPermission(getMapValueAsString(
                 currentAceDefinition, ACE_CONFIG_PROPERTY_PERMISSION));
+        // kept for backward compatibility reasons
         tmpAclBean.setRepGlob((String) currentAceDefinition.get(ACE_CONFIG_PROPERTY_GLOB));
+        
+        tmpAclBean.setRestrictions(currentAceDefinition, tmpAclBean);
```

Comment:
This is passing tmpAclBean to method setRestrictions that is called on same tmpAclBean. Would it be better here to have 
tmpAclBean.loadRestrictionsFrom(currentAceDefinition) ?